styles.css

```diff
@@ -225,6 +225,33 @@ hr {
   padding-bottom: 24px;
 }
 
+/* about section */
+
+.about {
+  margin-bottom: -5px;
+}
+
+.about h2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  color: var(--primary);
```

Comment:
Properties like `color`, `font-family` are usually defined at the tag level, this way we repeat less code